### PR TITLE
Move comments activities to the new API

### PR DESCRIPTION
**Pull request by nickvergessen**
### Steps
1. Comment on a file mentioning a user
2. Go to the activity app and see that the filter in the sidebar and the activity show the icon
3. The user mention in the message should have the avatar next to it.

Follow up to #2092

### CODE REVIEW

**Comment by MorrisJobke** [comment]
Tested and works 👍 


@rullzer @icewind1991 @LukasReschke Please review?

**Comment by MorrisJobke** [comment]
💥 
```
1) OCA\Comments\Tests\Unit\AppInfo\ApplicationTest::test
OCP\AppFramework\QueryException: Could not resolve OCA\Comments\Activity\Extension! Class OCA\Comments\Activity\Extension does not exist
/drone/src/github.com/nextcloud/server/lib/private/AppFramework/Utility/SimpleContainer.php:102
/drone/src/github.com/nextcloud/server/lib/private/AppFramework/Utility/SimpleContainer.php:117
/drone/src/github.com/nextcloud/server/lib/private/AppFramework/DependencyInjection/DIContainer.php:540
/drone/src/github.com/nextcloud/server/apps/comments/tests/Unit/AppInfo/ApplicationTest.php:61

```

**Comment by MorrisJobke** [comment]
@rullzer @icewind1991 @LukasReschke Please review ;)

File: apps/files/lib/Activity/Provider.php [hunk 1]
```diff
@@ -1,22 +1,21 @@
 <?php
 /**
- * @copyright Copyright (c) 2016, ownCloud, Inc.
+ * @copyright Copyright (c) 2016 Joas Schilling <coding@schilljs.com>
  *
- * @author Joas Schilling <coding@schilljs.com>
+ * @license GNU AGPL version 3 or any later version
```
Comment (rullzer) [inline apps/files/lib/Activity/Provider.php]: We can't just change this

File: lib/public/Activity/IFilter.php [hunk 2]
```diff
@@ -1,24 +1,21 @@
 <?php
 /**
- * @copyright Copyright (c) 2016, ownCloud, Inc.
+ * @copyright Copyright (c) 2016 Joas Schilling <coding@schilljs.com>
  *
- * @author Joas Schilling <coding@schilljs.com>
- * @author Morris Jobke <hey@morrisjobke.de>
- * @author Thomas Müller <thomas.mueller@tmit.eu>
+ * @license GNU AGPL version 3 or any later version
```
Comment (rullzer) [inline lib/public/Activity/IFilter.php]: Same here...

**Comment by rullzer** [comment]
Would it be possible to get the test coverage up a bit? Or is that taken care of in any of the later PR's?

File: apps/files/lib/Activity/Provider.php (hunk 1 above)
Comment (nickvergessen) [inline apps/files/lib/Activity/Provider.php]: This is a new file, yes we can?

Comment (nickvergessen) [inline apps/files/lib/Activity/Provider.php]: Ah well, it's from the part1 PR in https://github.com/nextcloud/server/pull/2092
But still it's a new file that was just commited with wrong header

File: lib/public/Activity/IFilter.php (hunk 2 above)
Comment (nickvergessen) [inline lib/public/Activity/IFilter.php]: Ah well, it's from the part1 PR in https://github.com/nextcloud/server/pull/2092
But still it's a new file that was just commited with wrong header

**Comment by nickvergessen** [comment]
> Would it be possible to get the test coverage up a bit? Or is that taken care of in any of the later PR's?

Yeah, after the beta, currently its "done > perfect"

**Comment by rullzer** [comment]
Fair enough then LGTM

**Pull request merged**